infoschema/tables.go

```diff
@@ -278,14 +278,19 @@ var tableIDMap = map[string]int64{
 	TablePlacementPolicies:               autoid.InformationSchemaDBID + 79,
 }
 
+// columnInfo represents the basic column information of all kinds of INFORMATION_SCHEMA tables
 type columnInfo struct {
-	name      string
-	tp        byte
-	size      int
-	decimal   int
-	flag      uint
-	deflt     interface{}
-	comment   string
+	name string
+	// tp represent column type
+	tp      byte
+	size    int
+	decimal int
+	// flag represent NotNull, Unsigned, PriKey flags etc.
+	flag uint
+	// deflt represent default value
+	deflt   interface{}
+	comment string
+	// enumElems represent elements when the column type is enum
```

Comment:
Maybe more detailed, e.g. `enumElems` are all possible literal string values of an enum column

Reply:
ok, THKS